docs/conversion_examples_gallery/recording/spikeglx.rst

```diff
@@ -26,7 +26,4 @@ Convert spikeglx data to NWB using :py:class:`~neuroconv.datainterfaces.ecephys.
     >>> # Choose a path for saving the nwb file and run the conversion
     >>> nwbfile_path = f"{path_to_save_nwbfile}"
     >>> interface.run_conversion(nwbfile_path=nwbfile_path, metadata=metadata)
-    >>>
-    >>> # If the conversion was successful this should evaluate to ``True`` as the file was created.
-    >>> Path(nwbfile_path).is_file()
-    True
```

Comment:
I always thought this looked a little awkward to introduce what is basically an assertion statement into code that is meant to be easily shared and interpretable by novices to coding in general.

With this approach, we can save 3 lines per gallery and the output is just as (if not more) informative when the user themselves tries to run it on their data and file paths.

Reply:
That line was not meant to be informative to the user. It was just as a means to test that the conversion run OK for us (to alert ourselves that our documentation was not working with our code base). To be honest, I am not sure that what I did does a great job at that goal but is appropriate to ask about this new approach: does the doctest now fail if the conversion fails? What do you think?

I sort of agree with the virtue of your approach by the way. I think that hiding details from first time users is great (that's why I think making them modify metadata is noise). That said, you seem to justify it in terms of saving lines which I think is always the wrong metric to focus on (reducing number of lines). Is there something I am missing there?